h/streamer/nsq.py

```diff
@@ -79,10 +75,7 @@ def handle_message(message, topic_handlers=None):
     incoming messages and messages to be sent out over the websockets.
     """
     if topic_handlers is None:
```

Comment:
The `topic_handlers` argument should be made non-optional, since the function doesn't do anything useful without it.